README.md

```diff
@@ -53,9 +53,14 @@ This documentation provides a comprehensive overview of the UKEF Digital TradeFi
 6. Set UKEF TFM environment variables in your terminal: `UKEF_TFM_API_SYSTEM_KEY` and `UKEF_TFM_API_REPORTS_KEY`.
 7. Run `npm run env:copy` to copy your root .env file into all the individual projects that need it.
 8. Run `npm ci` in the root folder of the repository. (note: this will install dependencies for the entire project, including those specified in sub-packages. More details on this in the [npm workspaces](./doc/npm-workspaces.md) docs)
-9. Start your local environment with `npm run start`.
-10. Run migrations on the MSSQL Server database (see [SQL DB docs](./doc/sql-db.md#--run-migrations) for details)
-11. Create mock data by running `npm run load` from the root folder of the repository. This should generate mocks in your database (both Mongo and MSSQL). (for more details on what this does please see [utils docs](./utils/README.md))
+9. Start your local environment with `docker-compose up --build`.
+10. Create mock data
+    - If not working on utilisation reports:
+      1. Run migrations on the MSSQL Server database by running `npm run db:migrate` in the `libs/common` directory (see [SQL DB docs](./doc/sql-db.md#--run-migrations) for details)
+      2. Create mock data by running `npm run load` from the root folder of the repository. This should generate mocks in your database (both Mongo and MSSQL). (for more details on what this does please see [utils docs](./utils/README.md)).
```

Comment:
I think `npm run load` should be it's own step after no.9

Reply:
I disagree because if working on utilisation reports, you will waste a step as you only need to run npm run load:init

Reply:
But `npm run load` is required for everything else

If the existing utilisation report migration scripts run `npm run load` then whilst it's not optimal, I think it's less optimal to have `npm run load` requirement nested in "if X if Y" style documentation